qa/L0_tftrt_optimization/test.sh

```diff
@@ -178,6 +178,47 @@ for MODEL in \
         RET=1
     fi
 
+    set -e
+
+    kill $SERVER_PID
+    wait $SERVER_PID
+done
+
+for MODEL in \
```

Comment:
So it was failing because the TensorRT optimization was not actually enabled?

Reply:
Just confirmed with Guan, this was not the case. We see segfaults even with the current case.